server/src/api/v1.py

```diff
@@ -168,7 +191,10 @@ def check_token_permissions(
         auth_token, secret_key, queue, priority
     )
     queue_allowed = check_token_queue(auth_token, secret_key, queue)
-    return priority_allowed and queue_allowed
+    reservation_time_allowed = check_token_reservation_timeout(
```

Comment:
I would say this permission check only makes sense if this is a reservation job, right?

Reply:
Yeah, but if the job isn't a reservation job, it will allow it.